src/server/lib/okta/registration.ts

```diff
@@ -0,0 +1,33 @@
+import { CreateUserRequestOptions } from '@okta/okta-sdk-nodejs/src/types/models/CreateUserRequest';
+import { oktaClient } from '@/server/lib/okta/client';
+import { sendOktaActivationEmail } from '@/server/lib/idapi/user';
+
+export const register = (
+  email: string,
+  ip: string,
+  returnUrl?: string,
+  refViewId?: string,
+  ref?: string,
+  activate = false,
+  sendEmail = false,
```

Comment:
if we always want these to be `false`, as i can't see a use case right now where we want to override these, i suggest that they shouldn't be parameters and be hardcoded as `false` where they are used

Reply:
I have extracted these into variables instead of arguments